src/main/java/com/kmbl/InventoryManagementService/models/ResponseObject.java

```diff
@@ -0,0 +1,12 @@
+package com.kmbl.InventoryManagementService.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
```

Comment:
Don't need to response object individually , add all the items in the same ResponseBody only